packages/cors/lib/cors.js

```diff
@@ -78,9 +84,15 @@ module.exports = (options, caches=new Caches()) => {
     }
     // Add CORS and AMP CORS headers
     response.setHeader('Access-Control-Allow-Origin', originHeaders.origin || sourceOrigin);
+    const headersToExpose = [];
     if (options.enableAmpRedirectTo) {
-      response.setHeader('Access-Control-Expose-Headers', 'AMP-Redirect-To');
+      headersToExpose.push('AMP-Redirect-To');
+    }
+    if (options.email) {
```

Comment:
Maybe add additional check to ensure it's a valid request, e.g. `if (options.email && sourceOrigin.contains('@')) {`

Reply:
Not sure if there's match value in this check...and I don't want to go down the email regex rabbit hole.